### PR TITLE
New vSphere cluster/node-template docs

### PR DESCRIPTION
**Pull request by janeczku**

### CODE REVIEW

**Comment by kylerome** [comment]
looks good to me

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/node-pools/vsphere/_index.md [hunk 1]
```diff
@@ -7,68 +7,139 @@ aliases:
 ---
 Use {{< product >}} to create a Kubernetes cluster in vSphere.
 
-### Prerequisites
+## Introduction
 
-Before setting up a cluster in vSphere, you must ensure that you have a user in vSphere with sufficient permissions to create a Rancher cluster.
+When creating a vSphere cluster Rancher first provisions the specified amount of virtual machines by communicating with the vCenter API. Then it installs Kubernetes on top of them. A vSphere cluster may consist of multiple groups of VMs with distinct properties, such as the amount of memory or the number of vCPUs. This allows for fine-grained controll over the sizing of nodes for the data, control and worker plane respectively.
```
Comment (MBishop17) [inline content/rancher/v2.x/en/cluster-provisioning/rke-clusters/node-pools/vsphere/_index.md]: Add a comma: "When creating a vSphere cluster, Rancher"

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/node-pools/vsphere/_index.md [hunk 2]
```diff
@@ -7,68 +7,139 @@ aliases:
...
+
+> *Note:*
```
Comment (MBishop17) [inline content/rancher/v2.x/en/cluster-provisioning/rke-clusters/node-pools/vsphere/_index.md]: Change italics to bold: "**Note:**"

**Comment by MBishop17** [comment]
LGTM too, although I have a few style/grammar changes I want to merge over the top of this: #599

**Pull request merged**